tests/test_master.py

```diff
@@ -108,10 +123,26 @@ def test_correlation_matrix_dont_mix_up_partly_matching_keys():
     assert result[COLUMN1][COLUMN2] == 1
 
 
+def test_train_model_accepts_dataset():
+    dataset = load_dataset()
+    pipe = pipeline.make_pipeline(StandardScaler(), GaussianNB(priors=None))
+
+    client = create_base_mock_client()
+    client.get_results.return_value = [dataset]
+    result = master.train_model(client, None, pipe, FEATURES, TARGET, IDENTIFIER_KEYS)
+
+    print(result)
+    assert True
```

Comment:
`assert True` 🎉

Reply:
Are you mocking me ;P  I'm interested in whether the function succeeds at all. This test is very important for debugging. I could also check the result of the model but I'd say that value (mean absolute error at the moment) is too specific and meaningless.